registry/remote/example_test.go

```diff
@@ -76,6 +77,43 @@ var (
 		{exampleSBoMManifestDescriptor},      // page 0
 		{exampleSignatureManifestDescriptor}, // page 1
 	}
+	formatZipManifest, _ = json.Marshal(artifactspec.Manifest{
+		MediaType:    artifactspec.MediaTypeArtifactManifest,
+		ArtifactType: "format/zip"})
+	formatZipManifestDescriptor = artifactspec.Descriptor{
+		MediaType:    artifactspec.MediaTypeArtifactManifest,
+		ArtifactType: "format/zip",
+		Digest:       digest.FromBytes(formatZipManifest),
+		Size:         int64(len(formatZipManifest))}
+	formatRarManifest, _ = json.Marshal(artifactspec.Manifest{
+		MediaType:    artifactspec.MediaTypeArtifactManifest,
+		ArtifactType: "format/rar"})
+	formatRarManifestDescriptor = artifactspec.Descriptor{
+		MediaType:    artifactspec.MediaTypeArtifactManifest,
+		ArtifactType: "format/rar",
+		Digest:       digest.FromBytes(formatRarManifest),
+		Size:         int64(len(formatRarManifest))}
+	formatBinaryManifest, _ = json.Marshal(artifactspec.Manifest{
+		MediaType:    artifactspec.MediaTypeArtifactManifest,
+		ArtifactType: "format/binary"})
+	formatBinaryManifestDescriptor = artifactspec.Descriptor{
+		MediaType:    artifactspec.MediaTypeArtifactManifest,
+		ArtifactType: "format/binary",
+		Digest:       digest.FromBytes(formatBinaryManifest),
+		Size:         int64(len(formatBinaryManifest))}
+	formatManifest, _ = json.Marshal(artifactspec.Manifest{
+		MediaType:    artifactspec.MediaTypeArtifactManifest,
+		ArtifactType: "example/formats",
+		Blobs: []artifactspec.Descriptor{
```

Comment:
`Blobs` should not include manifests. You may refer to https://github.com/oras-project/artifacts-spec/blob/main/examples/net-monitor-image-signature.json#L4-L10 and https://github.com/oras-project/artifacts-spec/blob/main/examples/net-monitor-image-sbom.json#L4-L10.

Reply:
Changed so that blobs include non-manifest contents.